ProcessManager.py

```diff
@@ -66,47 +77,40 @@ def createProcess(self, name, command):
 
         self.processes[name] = RoboProcess(command)
 
-    # TODO: Do we really need this if we have createProcess?
-    def createThread(self):
+
+    def start(self,**name):
```

Comment:
The syntax `**name` takes in a variable number of arguments indexable by keyword:
``` python
def send_msg(**kwargs):
    print(kwargs["test"])

send_msg(test="Hi")
```
ie kwargs is a dictionary.
I think we want more of a bunch of strings to be passed in:
``` python
...
    def start(self, *args):
        for name in args:
             self.processes[name] ... # start the process
...

manager.start("sleeper", "crash", "somethingelse")
```
But first check if the length of `args` is not 0 before looping over it.